tests/cli/test_compose_ext4-filesystem.sh

```diff
@@ -32,6 +45,11 @@ rlJournalStart
 
     rlPhaseStartCleanup
         rlRun -t -c "$CLI compose delete $UUID"
+        if [[ -n $SUBSCRIPTION_HOSTNAME && -n $SUBSCRIPTION_USERNAME && -n $SUBSCRIPTION_PASSWORD ]]; then
+            rlRun -t -c "subscription-manager unregister"
+            rlRun -t -c "mv -f /etc/yum.repos.d/disabled/*.repo /etc/yum.repos.d/"
+            rlRun -t -c "rm -rf /etc/yum.repos.d/disabled"
```

Comment:
Same here, wrt restoring. You can overwrite the entire directory in a single operation. Probably it would be easier if both locations are on the same level in the file system.